ToolBox/Benzinga/BenzingaNewsDataConverter.cs

```diff
@@ -0,0 +1,405 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using Newtonsoft.Json;
+using QuantConnect.Configuration;
+using QuantConnect.Data.Custom.Benzinga;
+using QuantConnect.Logging;
+using QuantConnect.Util;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.IO;
+using System.Xml;
+using QuantConnect.Interfaces;
+using QuantConnect.Data.Auxiliary;
+using Newtonsoft.Json.Linq;
+
+namespace QuantConnect.ToolBox.Benzinga
+{
+    /// <summary>
+    /// Converts Historical data in RSS format to
+    /// </summary>
+    public class BenzingaNewsDataConverter
+    {
+        private readonly DirectoryInfo _sourceDirectory;
+        private readonly DirectoryInfo _destinationDirectory;
+        private readonly DirectoryInfo _processedFilesDirectory;
+        private readonly IMapFileProvider _mapFileProvider;
+        private readonly MapFileResolver _mapFileResolver;
```

Comment:
nit -
`using System.Xml;` and `private readonly MapFileResolver _mapFileResolver;` are not being used